### PR TITLE
Print ebinds in FQ and fix crash with nested quantifiers

### PR DESCRIPTION
**Pull request by atondwal**

### CODE REVIEW

**Comment by ranjitjhala** [comment]
Please don’t merge into develop until and unless the LH test also work.

**Comment by atondwal** [comment]
It fixes the LH crashes (make one of the tests crash -> SAFE and another from crash -> UNSAFE), but I was going to hold off until you reviewed the LH PR and merge them together anyways

**Comment by ranjitjhala** [comment]
:thumbsup:

File: src/Language/Fixpoint/Solver/Solve.hs [hunk 1]
```diff
@@ -301,25 +301,54 @@ partitionInfo (i, fi)
...
+  where
     be             = F.bs si
     c              = Misc.safeLookup "solveEbinds" cid (F.cm si)
 
 
-exElim :: F.SEnv (F.BindId, F.Sort) -> F.BindId -> F.Pred -> F.Pred 
-exElim env xi p = F.notracepp msg (F.pExist yts p) 
-  where 
+exElim :: F.SEnv (F.BindId, F.Sort) -> F.BindId -> F.Symbol -> F.Pred -> F.Pred
+exElim env xi x p = F.notracepp msg (F.pExist yts (clearExist x p))
```
Comment (ranjitjhala) [inline src/Language/Fixpoint/Solver/Solve.hs:320]: What is this `clearExist` for?

**Comment by ranjitjhala** [comment]
Ok, am going to merge and then UNDO the `clearExists`.

**Pull request merged**